Banshee.sln

```diff
@@ -1,649 +1,659 @@
-﻿

```

Comment:
Would you mind un-fixing the line endings on this file? - I think the whitespace diff showed just a couple of projects being added to both Debug and Release build configurations.

You may have sane git attributes or `core.autocrlf` set - I can never figure out which, line endings in git are a bit of black magic - unfortunately the repository at this point doesn't have this commit which should get us all on the same page: b0d226fe0ddc651b6cbc54e1c02265e7f91ef26b (it's in feature/fixes)

If you like I'll merge that commit into master and then if you rebase on top of that it should clean up the diff?

Reply:
Yep, let's do that.

Reply:
done :+1: #31